www/app/components/dashboard/views/pod.html

```diff
@@ -21,7 +21,7 @@
           <tr>
             <td class="name">Status</td>
             <td class="value">
-              {{pod.currentState.status}} on <a ng-href="/#/dashboard/groups/host/selector/host={{pod.currentState.host}}">{{pod.currentState.host}}</a>
+              {{pod.currentState.status}} on <a ng-href="#/dashboard/groups/host/selector/host={{pod.currentState.host}}">{{pod.currentState.host}}</a>
```

Comment:
Please convert to v1beta3 syntax. We're eliminating v1beta1 within the next week or two.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go